horizons/component/tradepostcomponent.py

```diff
@@ -171,8 +173,8 @@ def sell(self, res, amount, price, player_id):
 			remnant = self.get_inventory().alter(res, -amount)
 			assert remnant == 0
 			self.trade_history.append((Scheduler().cur_tick, player_id, res, -amount, price))
-			self.sell_history[ Scheduler().cur_tick ] = (res, amount, price)
-			self.total_income += amount*price
```

Comment:
This has nothing to do with the rest, doesn't it? Please keep you commits focused on the task they shall do.